README.md

```diff
@@ -74,6 +78,9 @@ Send-MailKitMessage @Parameters
 ```
 
 # Releases
+### 3.2.0-preview2
+* Added parameter for message priority
+
```

Comment:
This change can be removed, a couple other updates need to be made before release 3.2.0-preview2